### DISCUSSION TITLE
Cannot Sign in

### DISCUSSION THREAD

**Discussion by konda-sumanayana** [community/community#142663] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi,

I recently had to reset my laptop and lost recovery codes and authenticator with that. The laptop had an ssh with the github account and I am now completely locked out. I have the email and all the details associated with that account and would love for a way for me to access my account back. Is there anything I could do?

**Comment by moranenzo** [comment]
<p>Hi,</p>

<p>I'm sorry to hear about the trouble you're experiencing with your GitHub account. If you've lost your recovery codes and authenticator, and you no longer have access to the SSH key that was associated with your account, here are some steps you can take to try to regain access:</p>

<p>First, attempt to reset your password. Go to the GitHub login page and click on "Forgot password?" Enter your email address and follow the instructions in the email you receive to reset your password.</p>

<p>If resetting your password doesn’t work because of the two-factor authentication (2FA) requirement, you can use GitHub's recovery options. Visit the GitHub support page for account recovery, where they provide a process for situations like yours.</p>

<p>If you're still unable to access your account after trying the above steps, I recommend reaching out to GitHub Support directly. Provide them with as much information as possible about your account (such as your username, associated email, and any other details) to help them assist you effectively. You can contact them at <a href="https://support.github.com/contact" target="_blank">GitHub Support</a>.</p>

<p>Be prepared to answer any security questions that might come up during the recovery process.</p>

<p>Unfortunately, without access to your recovery codes or authenticator app, it might take some time to regain access, but GitHub Support should be able to assist you.</p>

<p>Best of luck!</p>

**Comment by ebndev** [comment]
Hi @konda-sumanayana,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
